docs/model-dev-guide/api-guides/apis-howto/api-keras-ug.rst

```diff
@@ -7,129 +7,155 @@
 
-******************
- Define the Model
-******************
+*****************
+ Build the Model
+*****************
 
-Users are required wrap their model prior to compiling it using :meth:`self.context.wrap_model
-<determined.keras.TFKerasTrialContext.wrap_model>`. This is typically done inside
-:meth:`~determined.keras.TFKerasTrial.build_model`.
+Building a distributed-capable model is easy in keras; you just need to wrap your model building and
+compiling in the ``strategy.scope()``.  See the `TensorFlow documentation
+<https://www.tensorflow.org/tutorials/distribute/keras
+#create_the_model_and_instantiate_the_optimizer>`__ for more detail.
 
-******************************************
- Customize Calling Model Fitting Function
-******************************************
+.. code:: python
 
-The :class:`~determined.keras.TFKerasTrial` interface allows the user to configure how ``model.fit``
-is called by calling :meth:`self.context.configure_fit()
-<determined.keras.TFKerasTrialContext.configure_fit>`.
+   def main(core_context, strategy):
+       with strategy.scope():
+           model = my_build_model()
+           model.compile(...)
 
-***************
- Checkpointing
-***************
+***********************************
+ Create the ``DeterminedCallback``
+***********************************
 
-A checkpoint includes the model definition (Python source code), experiment configuration file,
-network architecture, and the values of the model's parameters (i.e., weights) and hyperparameters.
-When using a stateful optimizer during training, checkpoints will also include the state of the
-optimizer (i.e., learning rate). You can also embed arbitrary metadata in checkpoints via a
-:ref:`Python SDK <store-checkpoint-metadata>`.
+The :class:`~determined.keras.DeterminedCallback` will automatically integrate your training with
+the Determined cluster.  It reports train and test metrics, it reports progress, it saves
+checkpoints, and it uploads them to checkpoint storage.  It also handles preemption signals from the
+Determined master (such as if you pause your experiment), shutting down training, then it restores
+training from where it left off when the experiment continues.
 
-TensorFlow Keras trials are checkpointed to a file named ``determined-keras-model.h5`` using
-``tf.keras.models.save_model``. You can learn more from the `TF Keras docs
-<https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/keras/models/save_model>`__.
+The ``DeterminedCallback`` has only three required inputs:
+   -  the ``core_context`` you already created
+   -  a ``checkpoint`` UUID to start training from, or ``None``.
+   -  a ``continue_id`` used to decide how to treat the checkpoint.
 
-***********
- Callbacks
-***********
+In training jobs, a value for ``checkpoint`` should be obtained from
+``det.get_cluster_info().latest_checkpoint``, which will automatically be populated with the latest
+checkpoint saved by this trial, if there is one.
```

Comment:
is it "a value" or "the value"?

e.g.,

In training jobs, the value for ``checkpoint`` should be retrieved from ``det.get_cluster_info().latest_checkpoint``. This field will automatically contain the most recent checkpoint saved by the trial, if one exists.

Reply:
It is "a value", because users can put whatever checkpoint they want to restore from in there.

For instance, a user might want to have a starting checkpoint in mind, but also support pausing and resuming, they might want to use:

```
info.latest_checkpoint or my_starting_checkpoint
```

as the value.  I'll just elaborate on this sentence, it probably belongs in the user guide.